### PR TITLE
Update install.cmd

### PR DESCRIPTION
**Pull request by gisphm**
1. Reduce duplicate code in the previous two for loops and copy bash_completion.sh to where it should be installed as the makefile describes.
2. Those commands in file `need_git_repo` actually don't need `is-git-repo`.

### CODE REVIEW

**Comment by qw3rtman** [comment]
Has this been tested?

**Comment by gisphm** [comment]
Yes, I am using it on my windows to update `git-extras`.

**Comment by qw3rtman** [comment]
@gisphm, what Git emulator are you using (`cygwin`, Git for Windows, `msysgit`)?

**Comment by gisphm** [comment]
Git for windows 2.6.4 64bit

**Comment by qw3rtman** [comment]
[ ] `cygwin`
[X] Git for Windows (@gisphm)
[ ] `msysgit`

@spacewander, do you still have access to a Windows computer with `msysgit`?

**Comment by gisphm** [comment]
I have to say that this script is written for `Git for Windows`. It does not work with `msysgit` or `cygwin`.

File: install.cmd [hunk 1]
```diff
@@ -8,25 +8,19 @@ IF NOT "%~1"=="" SET PREFIX="%~1"
...
+IF NOT EXIST %PREFIX%\etc\bash_completion.d MKDIR %PREFIX%\etc\bash_completion.d
+COPY %GITEXTRAS%\etc\bash_completion.sh	%PREFIX%\etc\bash_completion.d\git-extras
 
 FOR /R %GITEXTRAS%\bin %%i in (*.*) DO (
 	ECHO #!/usr/bin/env bash > %PREFIX%\bin\%%~ni
 	TYPE %GITEXTRAS%\helper\reset-env >> %PREFIX%\bin\%%~ni
 	TYPE %GITEXTRAS%\helper\git-extra-utility >> %PREFIX%\bin\%%~ni
-	TYPE %GITEXTRAS%\helper\is-git-repo >> %PREFIX%\bin\%%~ni
+	>NUL FINDSTR /C:%%~ni %GITEXTRAS%\need_git_repo || (
+		TYPE %GITEXTRAS%\helper\is-git-repo >> %PREFIX%\bin\%%~ni
```
Comment (JanSchulz) [inline install.cmd]: It took me a while to understand what this does:

```
 +     >NUL FINDSTR /C:%%~ni %GITEXTRAS%\need_git_repo || (
 +		TYPE %GITEXTRAS%\helper\is-git-repo >> %PREFIX%\bin\%%~ni
 +	)
``` 

`>NUL` prevents output from the findstr. Funny that you can write the pipe before the actual command...

Could probably easier to understand if this is refactored to 

```
FINDSTR /C:%%~ni %GITEXTRAS%\need_git_repo >NUL 
if %errorlevel% == 0 (...)
```

**Comment by JanSchulz** [comment]
This needs a rebase... :-)

**Comment by gisphm** [comment]
:sweat: @JanSchulz I am not good at batch programming...

**Pull request closed without merging**